test/jasmine/tests/sunburst_test.js

```diff
@@ -1345,7 +1345,7 @@ describe('Test sunburst texttemplate without `values` should work at root level:
         ['path: %{currentPath}', ['path: /', 'path: Eve/', 'path: Eve/', 'path: Eve/', 'path: Eve/', 'path: Eve', 'path: Eve/Seth', 'path: Eve/Seth/', 'path: Eve/Awan/']],
         ['%{percentRoot} of %{root}', ['100% of Eve', '33% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '17% of Eve']],
         ['%{percentEntry} of %{entry}', ['100% of Eve', '33% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '17% of Eve']],
-        ['%{percentParent} of %{parent}', ['%{percentParent} of %{parent}', '100% of Seth', '33% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '50% of Seth', '100% of Awan']],
+        ['%{percentParent} of %{parent}', ['100% of Eve', '100% of Seth', '33% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '17% of Eve', '50% of Seth', '100% of Awan']],
```

Comment:
Wait. Is this the `root` node sector? You're saying the parent of the root node is itself?

Reply:
Good call. Fixed in https://github.com/plotly/plotly.js/pull/4219/commits/54360be4be39d7d77a8f88fb147909efa6b037d4.

Reply:
I don't get it. After https://github.com/plotly/plotly.js/commit/54360be4be39d7d77a8f88fb147909efa6b037d4, looks like `%{parent}` is now `''`, but `%{percentParent}` is still `100%` for the root node. 

Can we have `'%{percentParent} of %{parent}'` resolve as `'%{percentParent} of %{parent}'` for the root node (i.e. the `percentParent` and `parent` keys are not defined) like before?

I know this is not ideal, but I don't like filling in `'%{percentParent}` and `%{parent}` with _wrong_ value for the root node just to make the template string look better.

Reply:
Done in https://github.com/plotly/plotly.js/pull/4219/commits/7155917f2dd0f4d440abb0fc89f86ff77c88439a.